docs/src/examples/components/Tooltip/Types/TooltipContentWrapperExample.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Button, Flex, Tooltip } from '@stardust-ui/react'
+
+const TooltipContentWrapperExample = () => {
+  const plainContentStyle = {
+    zIndex: 1000,
+    padding: 5,
+  }
+
+  return (
+    <Flex gap="gap.smaller">
+      <Tooltip content={<p style={plainContentStyle}>Plain tooltip content rendered 'as is'.</p>}>
```

Comment:
Do we really want to pay attention to this behavior? We previously had discussion and decision was to use `Popper` content for positioning purposes

Reply:
I just copy the examples we had in the Popup component, but you are right, we don't want to advertise this. Will remove it!